lib/generators.js

```diff
@@ -237,6 +237,10 @@ function _generatePaths(generated_routes, tags, customBlueprintMaps) {
         _.forEach(routes, function (route) {
 
             var tag = _.find(tags, {identity: identity});
+          
+            if (route.tag)  { // allow for overide of tags from route configuration - useful for standalone actions
+              tag = route.tag;
+            }
```

Comment:
instead of this, why not just update line 239 to look like so:

```js
var tag = route.tag || _.find(tags, {identity: identity});
```

Reply:
Sure, I only wanted the change to be as clear as possible. Will do.